### ISSUE TITLE
Use USB specific ESD protection package

### ISSUE THREAD

**Issue by warasilapm** [opulo-inc/lumenpnp#706]
# Version Number
Rev5b0

# Bugfix or Enhancement
Both?

# Description
The USB lines to/from the ESD diode chosen (SRV05-4) is routed a little awkwardly; I think this is a result of the four line configuration of the part chosen.

# Suggested Solution
Using something like the [USBLC6-2](https://www.st.com/en/protections-and-emi-filters/usblc6-2.html) much simplifies this routing.
![image](https://github.com/opulo-inc/lumenpnp/assets/37279515/f6023e22-3695-4c59-8e61-9891c0d64ecf)

**Comment by dan-gies** [comment]
I agree that the routing path on the current part is a little awkward.

@warasilapm note that the suggested part should be routed to only one of the two pins (e.g. 3 or 4) and pass under the part to maintain signal integrity. From the datasheet, ST shows one trace of the diff-pair passing under the part, not through it.

![image](https://github.com/opulo-inc/lumenpnp/assets/108100309/1a5917a6-858c-4b57-81c2-d56a7ce244df)

**Comment by warasilapm** [comment]
> I agree that the routing path on the current part is a little awkward.
> 
> 
> 
> @warasilapm note that the suggested part should be routed to only one of the two pins (e.g. 3 or 4) and pass under the part to maintain signal integrity. From the datasheet, ST shows one trace of the diff-pair passing under the part, not through it.
> 
> 
> 
> ![image](https://github.com/opulo-inc/lumenpnp/assets/108100309/1a5917a6-858c-4b57-81c2-d56a7ce244df)
> 
> 

I looked at that part of the datasheet; that is referring to the VBUS line specifically as it continues towards the rest of the board. If anything, it implies the  optional diode should follow the ESD IC in the circuit rather than branching off at the connector pin. IIRC the screen cap you took makes sense given a micro-b pinout but I didn't check.

The recommended layout reflects how I show the data lines.

**Comment by dan-gies** [comment]
@warasilapm you're right! I rushed to conclusions.

**Comment by warasilapm** [comment]
Image for reference:
![image](https://github.com/opulo-inc/lumenpnp/assets/37279515/7dde9208-0b94-4808-b4f1-8fd78b9428ce)

Btw I checked and it definitely did not make sense for micro USB. Who knows what they are thinking with that VBUS trace.

**Comment by dan-gies** [comment]
For both these parts (existing and new) it's a little weird that they pull the diff-pair so far apart, but I guess the ±Ω tolerance is big enough that it doesn't really matter for FS speeds. For 2.0 I'd think it'd be sub-par, but that's irrelevant here.
